### ISSUE TITLE
Some cutscenes remain stretched with ultrawide fix enabled

### ISSUE THREAD

**Issue by Darcrath** [Lyall/FFXVIFix#82]
When enabling the Fix Resolution setting some cutscenes (assuming prerendered) show as stretched. If these are prerendered and there is not a way to fix them is it possible to have a feature added that allows ultrawide during gameplay but disables it during cutscenes? I've attached a pair of examples.

![FFXVI_21_9_FMV](https://github.com/user-attachments/assets/8e238ca9-c4e9-4909-b381-d1af71261eb2)
![FFXVI_16_9_FMV](https://github.com/user-attachments/assets/f34a28ca-b386-46bd-a27f-6e33f64352f6)

**Comment by Lyall** [comment]
Can you post your `FFXVIFix.log` file please?

**Comment by Darcrath** [comment]
Yeah here you go, I have two. One from last night when I was messing around with different settings (FFXIVFix1.log) and one from today where I made sure most of the settings were returned to default. (FFXIVFix.log). Regardless of what settings I tweaked I still had the issue shown above. 

Let me know if I you need me to run it again with different settings and thanks for the look!

[FFXVIFix.log](https://github.com/user-attachments/files/17154984/FFXVIFix.log)
[FFXVIFix1.log](https://github.com/user-attachments/files/17154985/FFXVIFix1.log)

**Comment by Darcrath** [comment]
Yeah that's correct, 3440x1440 in borderless windowed. I can try switching to full screen to see if that changes it and reinstall the game along with the mod later to see if that works.

**Comment by Darcrath** [comment]
Running it at fullscreen did not result in any change. Pre-rendered cutscenes are still stretched but in game ones are not. I'll try reinstalling the game and the mod and update my findings here.

**Comment by Darcrath** [comment]
So I reinstalled the game and applied the mod from scratch using v0.8.4 and observed no change. I tried it in both fullscreen and windowed. I went back and downloaded and applied version 0.8.2 and found no change. Attempted with version 0.8.1 and found that pre-rendered cutscenes are now showing correctly with pillar boxes and no stretching. Version 0.8.1 is running in windowless mode.

Here are the logs from each attempt, I'm not sure if they will show you something helpful or not. There appears to be something different between version 0.8.1 and the subsequent versions that causes the pre-rendered cutscenes to show stretched for me.


[FFXVIFix_8.4_Windowless.log](https://github.com/user-attachments/files/17172996/FFXVIFix_8.4_Windowless.log)
[FFXVIFix_8.4_Fullscreen.log](https://github.com/user-attachments/files/17172998/FFXVIFix_8.4_Fullscreen.log)
[FFXVIFix_8.2.log](https://github.com/user-attachments/files/17173038/FFXVIFix_8.2.log)
[FFXVIFix_8.1.log](https://github.com/user-attachments/files/17173051/FFXVIFix_8.1.log)

**Comment by Lyall** [comment]
Hmm it doesn't make a lot sense at to why it wouldn't work for you but works for everyone else, including myself. If you give me some time I can try and come up with another place in memory to check if a movie is playing. I can then make a test build for you to try. Perhaps the code where it checks is not being executed on your machine for some bizarre reason.

**Comment by Sleix** [comment]
This is happening to me also. Downgrading to 0.8.1 shows the pillarboxes correctly.

[FFXVIFix.log](https://github.com/user-attachments/files/17177044/FFXVIFix.log)

**Comment by EuanFH** [comment]
just parroting i also had this issue and downgraded to 0.8.1 to have cut scenes display in the correct aspect. I'm using proton which could also be a contributing factor.

**Comment by Lyall** [comment]
> just parroting i also had this issue and downgraded to 0.8.1 to have cut scenes display in the correct aspect. I'm using proton which could also be a contributing factor.

Thanks for bumping this. It had slipped my mind and I've been really busy. 

I've changed the this test build to use a different area of code to check if a movie is playing. I'm making the assumption here that the issue lies with the check rather than the actual fix that adjust the movie width and so on. So if this does not work, then the issue probably lies with that part instead.

[FFXVIFix_v0.8.5_movies.zip](https://github.com/user-attachments/files/17327604/FFXVIFix_v0.8.5_movies.zip)

**Comment by EuanFH** [comment]
will give this a try to night when i find a cutscene will try current and this fix. will get back to you.

**Comment by EuanFH** [comment]
i have just been replacing the asi just confirming its not a ini config change as well

**Comment by Lyall** [comment]
Hmm, not sure what to do with it moving forward. It looks like the issue is with the hook that does the actual resizing of the movies and not the check to see if a movie is playing. I can't think of a great way to solve this problem unfortunately, I don't really know why current fix for movies does not work for yourself and some others when it does work fine for myself and others. I think maybe the best compromise I can come up with is to add the previous movie fix as an alternate option.

The downside of this approach is that it'll revert to the old behaviour of allowing you to see the game scene behind the resized movie. I unfortunately don't see a better way, as it took me many hours of research to reach the current method of fixing movies while retaining pillarboxing/letterboxing.

At least with an alternate option, there'll be something for those that the current movie fix does not work for.

**Comment by EuanFH** [comment]
Honestly mate I'm just glad you had a look at it probably a weird Linux thing. But it's also possibly other people are experiancing this but just not technical to open up a GitHub account to report it's broken. But I'm sure you know better than me the frequency of random people on the internet complaining about bugs hahaha.

If you decide to add the old version as a config option that would be great.

I'm happy to keep using 0.8.1 for now until you decide what you are or arnt doing.

**Comment by Lyall** [comment]
I've added it as an alternate option in v0.8.5. I'm sorry I can't come up with a better solution but at least it's something.

**Comment by Sleix** [comment]
Not sure if it helps, but I'm using an RX 7900 XTX (driver ver. 24.9.1). Is it possible it's an AMD or RDNA3 thing?
I'm using Windows 11, so it's not proton-specific.

**Comment by EuanFH** [comment]
I am also a using an AMD card 6800xt so it could possibly be an amd issue
